api/v2alpha1/go/pipelinespec/pipeline_spec.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.33.0
-// 	protoc        v3.17.3
+// 	protoc-gen-go v1.35.1
+// 	protoc        v4.25.3
```

Comment:
What command did you run that resulted in these versions getting bumped? They shouldn't get bumped if you're just going into the api folder and running `make golang` :thinking:

Reply:
might be this causing it:
```
$ pip install wheel setuptools protobuf==4.21.1 grpcio grpcio-tools
```

we should not use protobuf==4.21.1 there. What made you pick that version? Is that something I advised you to do? lol

Reply:
I actually did a conda install protobuf
`conda install -c conda-forge protobuf`
which installed this protoc version in my conda venv
```
protoc --version
libprotoc 25.3
```

I then went ahead and generated this `.pb.go` file, due to which protoc versions got bumped here. Any ideas on how I could regenerate the `.pb.go` without having these protoc versions updated?

Reply:
Resolved this by running `make golang` as per @gregsheremeta 's suggestion. Thanks!